Libraries/LibWebView/WebContentClient.cpp

```diff
@@ -500,14 +502,14 @@ void WebContentClient::did_expire_cookies_with_time_offset(AK::Duration offset)
     Application::cookie_jar().expire_cookies_with_time_offset(offset);
 }
 
-Messages::WebContentClient::DidRequestNewWebViewResponse WebContentClient::did_request_new_web_view(u64 page_id, Web::HTML::ActivateTab activate_tab, Web::HTML::WebViewHints hints, Optional<u64> page_index)
+void WebContentClient::did_request_new_web_view(u64 page_id, Web::HTML::ActivateTab activate_tab, Web::HTML::WebViewHints hints, Optional<u64> page_index, DidRequestNewWebView::Resolver resolve)
 {
     if (auto view = view_for_page_id(page_id); view.has_value()) {
-        if (view->on_new_web_view)
-            return view->on_new_web_view(activate_tab, hints, page_index);
+        if (view->on_new_web_view) {
+            return resolve(view->on_new_web_view(activate_tab, hints, page_index));
+        }
     }
-
-    return String {};
+    return resolve(String{});
```

Comment:
Here as well re: `return`, and many places below

Reply:
This was part of automated rewrite, in the final PR, these will be removed. `return resolve(value)` is as stated in other comment thread, useful for when we want to also exit the funciton - but as the last statement, it's just noise (as the function will return now anyway)